src/proper.erl

```diff
@@ -1173,6 +1260,60 @@ inner_test(RawTest, Opts) ->
 	false -> ShortResult
     end.
 
+%% @private
+-spec spawn_workers_and_get_result(
+        NumTests :: pos_integer(),
+        NumWorkers :: non_neg_integer(),
+        StrategyFun :: strategy_fun(),
+        SpawnFun :: fun((Args :: {Start, ToPass} | {node(), {Start, ToPass}})
+                          -> [{non_neg_integer(), non_neg_integer()}]),
+        NodeList :: [node()]) -> imm_result()
+            when Start :: non_neg_integer(), ToPass :: pos_integer().
+spawn_workers_and_get_result(NumTests, NumWorkers, StrategyFun, SpawnFun, NodeList) ->
+    TestsPerWorker = StrategyFun(NumTests, NumWorkers),
+    _ = maybe_start_cover_server(NodeList),
+    WorkerList =
+        case NodeList of
+            [] -> lists:map(SpawnFun, TestsPerWorker);
+            _ -> lists:map(SpawnFun, NodeList)
+        end,
+    InitialResult = #pass{samples = [], printers = [], actions = []},
+    AggregatedImmResult = aggregate_imm_result(WorkerList, InitialResult),
+    ok = maybe_stop_cover_server(NodeList),
+    AggregatedImmResult.
+
+%% @private
+%% @doc Runs PropEr in parallel mode, through the use of workers to perform the tests.
+%% Under this mode, PropEr needs information whether a property is pure or impure,
+%% and this information is passed via an option.
+%% When testing impure properties, PropEr will start a node for every worker that will be
+%% spawned in order to avoid test collisions between them.
+-spec parallel_perform(test(), opts()) -> imm_result().
+parallel_perform(Test, #opts{property_type = pure, numtests = NumTests,
+                             numworkers = NumWorkers, strategy_fun = StrategyFun} = Opts) ->
+    _ = maybe_start_cover_server([]),
```

Comment:
This line / call is not needed here; it will be done by `spawn_workers_and_get_result`.  (Or am I missing something?)

Reply:
Oh no, that was totally my mistake, thanks for pointing it out!

Reply:
Amended the last commit to remove the line.